templates/index.html

```diff
@@ -102,8 +102,33 @@ <h5 style="margin-bottom: 5%;">Equation Applied</h5>
             >
             </div>
         </div>
+        <div id="graph-select" style='margin-bottom: 5;'>
+            <select 
+                name="x-values-input" 
+                id="x-values-input"
+                class="custom-select"
+                style="width:15%"
+                >
+                <option id='x_active' value='' disabled selected >Select X Value</option>
```

Comment:
I know we added the x_active id's here but since they aren't used, don't think we need them.

Although, it could potentially be an easier way to get to that element when running tests

Reply:
I would say more reliable than easy. We could still get the element via an XPATH, it just might break easier and need to be updated more in the future after new code changes affecting it XPATH